.github/workflows/add-labels.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Add Labels'
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  add-labels:
+    if: ${{ !github.event.issue.pull_request }}
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Run ./.github/scripts/add-labels.py
+        run: python3 ./.github/scripts/add-labels.py
```

Comment:
This can easily be done with an inline powershell script. For an example see here: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5598/files#diff-ddf2bab74923dcd43f9f8d05e50bb1adc9ee5366a394d151d1f04a54a6079486R113-R144

I won't approve with a dependency on python. I firmly believe powershell is more useful in a .NET environment.

Reply:
We already having python scripts https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/build/sanitycheck.py

Reply:
Thanks for bringing the existing python to my attention. It doesn't change my stance though. My concern is maintenance of this rpeo, others may do as they please. If you want to stick with python consider spinning this up as a github action and referencing it here so the implementation is hidden. That is a better solution for achieving reusability of actions if that is your goal.